lib/postDecorator.js

```diff
@@ -135,17 +121,8 @@ function change() {
     },
     type: 'CHANGE_RESPONSE',
     getResponseBody() {
-      const ret = {}
-      if (this.body) {
-        ret.body = this.body
-      }
-      if (this.statusCode) {
-        ret.statusCode = this.statusCode
-      }
-      if (this.headers) {
-        ret.headers = this.headers
-      }
-      return ret
+      const { body, statusCode, headers } = { ...this }
```

Comment:
Why not just
```
const { body, statusCode, headers } = this
```
Is there a reason for the spread operator of the `this`?

Reply:
Nope, fixed